.circleci/config.yml

```diff
@@ -26,6 +26,22 @@ jobs:
       - run: cp /tmp/builds/libgo_cosmwasm.so ./api
       - run: make test
       - run: make build-go
+  # TODO: only run this on master (and not on tags)
+  release:
+    docker:
+      - image: circleci/rust:1.41-stretch
+    steps:
+      - run: rustup toolchain install nightly
```

Comment:
Can we version pin the Rust version? Otherwise we get putentially new versions every day just because some compiler internals change

Reply:
Yup, actually I need to use circleci machine (to launch those sub dockers in build). And as of https://github.com/CosmWasm/go-cosmwasm/pull/51 we do use a pinned nightly build

Reply:
I will update install to install the proper pin the first time